### PR TITLE
Incorporate documentation feedback - Ceph RGW

### PR DESCRIPTION
**Pull request by fmount**

### CODE REVIEW

File: docs_user/modules/proc_migrating-the-rgw-backends.adoc [hunk 1]
```diff
@@ -115,6 +118,29 @@ rgw.rgw.cephstorage-2.krycit  cephstorage-2  172.17.3.81:8090  running (13s)
...
+
+. If `nftables` is used in the existing deployment, edit `/etc/nftables/tripleo-rules.nft`
+and add the following lines:
++
+----
+# 100 ceph_rgw {'dport': ['8080','8090']}
+add rule inet filter TRIPLEO_INPUT tcp dport { 8080,8090 } ct state new counter accept comment "100 ceph_rgw"
+----
+
+. Save the file.
```
Comment (klgill) [inline docs_user/modules/proc_migrating-the-rgw-backends.adoc:133]:
```suggestion
. Save the file.

```

File: docs_user/modules/proc_migrating-the-rgw-backends.adoc [hunk 2]
```diff
@@ -115,6 +118,29 @@ rgw.rgw.cephstorage-2.krycit  cephstorage-2  172.17.3.81:8090  running (13s)
...
+. Restart the `nftables` service:
++
+----
+$ sudo systemctl restart nftables
+----
+
+. Verify the rules have been applied:
```
Comment (klgill) [inline docs_user/modules/proc_migrating-the-rgw-backends.adoc:140]:
```suggestion
. Verify that the rules are applied:
```

File: docs_user/modules/proc_migrating-the-rgw-backends.adoc [hunk 3]
```diff
@@ -115,6 +118,29 @@ rgw.rgw.cephstorage-2.krycit  cephstorage-2  172.17.3.81:8090  running (13s)
 ----
 $ iptables -I INPUT -p tcp -m tcp --dport 8080 -m conntrack --ctstate NEW -m comment --comment "ceph rgw ingress" -j ACCEPT
 $ iptables -I INPUT -p tcp -m tcp --dport 8090 -m conntrack --ctstate NEW -m comment --comment "ceph rgw backends" -j ACCEPT
+$ sudo iptables-save
+$ sudo systmectl restart iptables
+----
+
+. If `nftables` is used in the existing deployment, edit `/etc/nftables/tripleo-rules.nft`
+and add the following lines:
```
Comment (klgill) [inline docs_user/modules/proc_migrating-the-rgw-backends.adoc:126]:
```suggestion
and add the following content:
```

**Review by klgill — CHANGES_REQUESTED** [review]
Minor edits.

File: docs_user/modules/proc_migrating-the-rgw-backends.adoc (hunk 2 above)
Comment (fmount) [reply docs_user/modules/proc_migrating-the-rgw-backends.adoc]: Done

**Review by jistr — APPROVED** [review]

**Review by klgill — APPROVED** [review]

**Pull request merged**